help/destinations/catalog/crm/hubspot.md

```diff
@@ -0,0 +1,188 @@
+
+## Prerequisites {#prerequisites}
+
+Refer to the sections below for any prerequisites that you need to set up in Experience Platform and [!DNL HubSpot] and for information that you must gather before working with the [!DNL HubSpot] destination.
+
+### Experience Platform prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL HubSpot] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Refer to Experience Platform documentation for [Audience Membership Details schema field group](/help/xdm/field-groups/profile/segmentation.md) if you need guidance on audience statuses.
+
+### Prerequisites for the [!DNL HubSpot] destination {#prerequisites-destination}
+
+Note the following prerequisites in order to export data from Platform to your [!DNL HubSpot] account:
+
+#### You must have a [!DNL HubSpot] account {#prerequisites-account}
+
+In order to export data from Platform to your [!DNL Hubspot] account you need to have a [!DNL HubSpot] account. If you do not have one already, visit the [Set up your HubSpot account](https://knowledge.hubspot.com/get-started/set-up-your-account) page and follow the guidance to register and create your account.
+
+#### Gather the [!DNL HubSpot] private app access token {#gather-credentials}
+
+You need your [!DNL HubSpot] **Access token** to allow the [!DNL HubSpot] destination to make API calls through your [!DNL HubSpot] private app within your [!DNL HubSpot] account. The **Access token** serves as the **Bearer token** when you [authenticate the destination](#authenticate).
+
+If you do not have a private app, follow the documentation to [Create a private app in [!DNL HubSpot]](https://developers.hubspot.com/docs/api/private-apps).
+
+>[!IMPORTANT]
+>
+> The private app should be assigned the scopes below:
+> `crm.objects.contacts.write`, `crm.objects.contacts.read`
+> `crm.schemas.contacts.write`, `crm.schemas.contacts.read`
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| `Bearer token` | The `Access token` of your [!DNL HubSpot] private app. <br>To obtain your [!DNL HubSpot] **Access token** follow the [!DNL HubSpot] documentation to [make API calls with your app's access token](https://developers.hubspot.com/docs/api/private-apps#make-api-calls-with-your-app-s-access-token). | `pat-na1-11223344-abcde-12345-9876-1234a1b23456` |
```

Comment:
Please use `Access token` and `Bearer token` instead of bold versions.  (example column)